include/deal.II/matrix_free/portable_matrix_free.h

```diff
@@ -471,11 +487,35 @@ namespace Portable
            const Quadrature<1>             &quad,
            const AdditionalData            &additional_data = AdditionalData());
 
+
+    /**
+     * Reinit with a list of DoFHandler objects.
+     */
+    template <typename IteratorFiltersType>
+    void
+    reinit(const Mapping<dim>                                   &mapping,
+           const std::vector<const DoFHandler<dim> *>           &dof_handler,
+           const std::vector<const AffineConstraints<Number> *> &constraints,
+           const Quadrature<1>                                  &quad,
+           const IteratorFiltersType &iterator_filter,
+           const AdditionalData      &additional_data = AdditionalData());
+
+    /**
+     * Same as above using Iterators::LocallyOwnedCell() as predicate.
+     */
+    void
+    reinit(const Mapping<dim>                                   &mapping,
+           const std::vector<const DoFHandler<dim> *>           &dof_handler,
+           const std::vector<const AffineConstraints<Number> *> &constraints,
+           const Quadrature<1>                                  &quad,
+           const AdditionalData &additional_data = AdditionalData());
+
     /**
      * Return the Data structure associated with @p color.
      */
     PrecomputedData
-    get_data(unsigned int color) const;
+    get_data(const unsigned int dof_handler_index,
+             const unsigned int color) const;
```

Comment:
@tjhei This is the only function which requires `dof_handler_index` when using a single a `DoFHandler`. Should we reorder the parameters and use a default value for `dof_handler_index`, like we do for the other functions?

Reply:
That's reasonable, sure.

Reply:
That's reasonable, sure.